src/vl-loader.js

```diff
@@ -0,0 +1,61 @@
+import { VlElement, define } from '/node_modules/vl-ui-core/dist/vl-core.js';
+
+/**
+ * VlLoader
+ * @class
+ * @classdesc De loader component is een animatie dat aangeeft dat een pagina of website aan het laden is.
+ * 
+ * @extends VlElement
+ * 
+ * @property {boolean} data-vl-light - Attribuut wordt gebruikt om een alternatieve rendering te bekomen voor op een te zorgen dat de radio getoond wordt als een block element en bijgevolg de breedte van de parent zal aannemen.
+ * @property {boolean} data-vl-text - Attribuut wordt gebruikt om een informatieve tekst te tonen tijdens het laden.
+ * @property {boolean} data-vl-single - Attribuut wordt gebruikt om aan te geven dat er geen tekst getoond mag worden.
```

Comment:
Waarom de term "single"? 
Waarom is er voor het al dan niet tonen van tekst een attribuut nodig, kan er dan niet gewoon een leeg slot/text attribuut meegegeven worden?

Reply:
Nee in combinatie met A11Y is het beter om het single attribuut te gebruiken. Single omdat dit ook bij radio en checkbox het attribuut is.

Reply:
Voor accessibility snap ik nog wel dat je de tekst zou verbergen maar wel in de DOM laat staan. 
Ik snap niet hoe de term "single" iets te maken heeft met het al dan niet tonen van tekst?

Reply:
Idem gebruik bij radio en checkbox waar single betekent dat alleen de checkbox of radio getoond wordt, dus zonder tekst. Hier zal dus alleen het laden icoon getoond worden en de bijhorende tekst zal uitsluitend voor A11Y 'getoond' worden aan screen readers.

Reply:
Ok, maar dat het overal kaka is wilt niet zeggen dat het overal kaka moet blijven, toch?